stages/test/test_mkdir.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/python3
+
+import contextlib
+import os
+import subprocess
+
+import pytest  # type: ignore
```

Comment:
Why do we need "type: ignore" in this test? It seems no of the other tests sets it and removing it seems to have no ill effect on "make lint"?

Reply:
Again locally lint complains

Reply:
It might be something local but it's proably ok